packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard-for-patient.spec.tsx

```diff
@@ -48,3 +56,54 @@ describe('Patient dashboard for HCP', () => {
     expect(secondaryHeader).toHaveTextContent('DashboardDailyTrendsGenerate report')
   })
 })
+
+describe('Team menu for patient', () => {
+  beforeAll(() => {
+    mockPatientLogin(monitoredPatientAsTeamMember)
+    jest.spyOn(TeamAPI, 'getTeams').mockResolvedValue([teamOne])
+    jest.spyOn(TeamAPI, 'joinTeam').mockResolvedValue()
+    jest.spyOn(TeamAPI, 'getTeamFromCode').mockResolvedValue(iTeamOne)
+  })
+
+  it('should close the dialog after clicking the join care team button with success message', async () => {
+    await act(async () => {
+      renderPage('/')
+    })
+
+    const badgeTeamMenu = screen.getByLabelText('Open team menu')
```

Comment:
I would encapsulate all of this code in a method with a name such as `checkJoinTeamDialog`